src/networks.js

```diff
@@ -0,0 +1,55 @@
+import env from './environment'
+
+const DEFAULT_IPFS_GATEWAY = 'https://ipfs.eth.aragon.network/ipfs'
+const DEFAULT_ORG_LOCATION = 'agreement3.aragonid.eth'
+
+const NETWORK_ENVIRONMENTS = new Map([
+  [
+    'ethereum',
+    {
+      chainId: 1,
+      legacyNetworkType: 'main',
+      ensAddress: '',
+      nodes: {
+        defaultEth: 'https://mainnet.eth.aragon.network/',
+      },
+    },
+  ],
+  [
+    'rinkeby',
+    {
+      chainId: 4,
+      legacyNetworkType: 'rinkeby',
+      ensAddress: '',
+      nodes: {
+        defaultEth: 'https://rinkeby.eth.aragon.network/',
+      },
+    },
+  ],
+  [
+    'rinkeby-staging',
+    {
+      chainId: 4,
+      legacyNetworkType: 'rinkeby',
+      ensAddress: '',
+      nodes: {
+        defaultEth: 'https://rinkeby.eth.aragon.network/',
+      },
+    },
+  ],
+])
+
+export function getNetworkEnvironment(environment) {
+  // If custom config supplied
+  if (environment.startsWith('{')) {
+    return JSON.parse(environment)
+  }
+
+  // Or use preset
+  return NETWORK_ENVIRONMENTS.get(environment)
+}
+
+export const orgLocation = env('ORG_LOCATION') || DEFAULT_ORG_LOCATION
+
+export const defaultIpfsGateway =
```

Comment:
After having to grapple with awkward dependency trees in the client, I wonder if we should keep this file as _pure_ configuration (without any imports or weird run-time side effects).

With the client, one of the changes I was going to do was move these configuration files into a `/configuration` folder, and maintain a `current-environment.js` that contains all the run-time selected defaults and settings.

Reply:
I like this idea, in terms of reusability should we expect others who fork the project to update this configuration file in place of individual environment variables?

Reply:
> I like this idea, in terms of reusability should we expect others who fork the project to update this configuration file in place of individual environment variables

Oh yes, I think that would be a great way forward! Having a quick little guide like, "go here, edit this, set up your Vercel account, push" would be awesome once we want others to fork!

Reply:
What would you guys think about having all of these coming from the env vars, so that the entire app environment could be swapped without having to edit any source file? And we could use a default `.env` to have an equivalent of `current-environment.js`.

Reply:
I think this is already the case though, and we will still want some sort of `current-environment.js` file to call and export the results of `env()`?

Reply:
Oh I see, I thought we were talking about not using `process.env` anymore, but it’s more about having a convenient place to find all the environment-imported values. Sounds good :+1:

Reply:
Ok, I made these changes, it's strictly build-time config now, I think it's fine and we shouldn't overthink it at this stage. Anything glaring let me know, else I'd like to get this in and move on :)